jax/scipy/stats/__init__.py

```diff
@@ -33,3 +33,4 @@
 from jax.scipy.stats import betabinom as betabinom
 from jax.scipy.stats import gennorm as gennorm
 from jax._src.scipy.stats.kde import gaussian_kde as gaussian_kde
+from jax._src.scipy.stats import _core as _core
```

Comment:
`_core` is meant to be an internal-only submodule... instead we should import `mode` directly into this namespace (`jax.scipy.stats.mode` like `scipy.stats.mode`)